<launch>
    <node pkg="spirecv_ros" type="mbzirc_usv_vessel_detection" name="mbzirc_usv_vessel_detection" output="screen">
        <param name="camera_parameters" type="string" value="$(find spirecv_ros)/camera_params/calib_webcam_640x480.yaml" />
        <param name="local_saving_path" type="string" value="$(find spirecv_ros)/video" />
    </node>

</launch>
